csv-extractor/csv_extractor.py

```diff
@@ -155,25 +152,46 @@ def process_csv_file(client, monitor, csv_path, existing_time_series):
                 target=_log_error, args=(client.datapoints.post_multi_time_series_datapoints, current_time_series)
             )
         )
-    end_time_1 = time.time()
-    logger.debug("Time taken to process file " + str(csv_path) + " " + str((end_time_1 - start_time)))
-    [t.start() for t in network_threads]
-    [t.join() for t in network_threads]
-    logger.debug("Time take to complete network requests & total time to ingest file  " + str(csv_path) + " " +
-                str(time.time()-end_time_1) + " " +
-                 str(time.time() - start_time))
 
-    return count_of_data_points, len(unique_external_ids)
+    return count_of_data_points, len(unique_external_ids), network_threads, csv_path
+
+
+def post_all_data(q, monitor, remaining_files_count):
```

Comment:
You should use black https://github.com/python/black on your code. It would make your code follow the python coding standard :)

Single char variables (except i) is not pleasant for us with diminishing eye-sight 😝